### DISCUSSION TITLE
Copilot is facing temporary errors!

### DISCUSSION THREAD

**Discussion by PrimeTheFirst** [community/community#10979] [Copilot Conversations] [answered]
I waited forever. I got a mail saying I got the technical preview. Then I installed the extension. Accepted telementery terms. Then took a example from the docs. But copilot says it's facing temporary errors when I hover over it. What do I do? I reinstalled copilot and reopened vscode. No avail.

**Comment by davecheney** [comment, marked as answer]
Thank you. We know that warning notification is too sensitive and we're actively fixing it so it fires less often. If you like, switch to the copilot-nightly build to get access to new changes as they happen. https://marketplace.visualstudio.com/items?itemName=GitHub.copilot-nightly

**Reply by PrimeTheFirst** [reply]
Thanks. I switched to nightly. Hope the fix comes soon.

**Reply by PrimeTheFirst** [reply]
How long will it take for the fix?

**Reply by davecheney** [reply]
It should be landing soon. Btw, the fix just makes the warning notification show up less often, it’s not actually complaining about a real problem, just randomly crying wolf.

**Reply by PrimeTheFirst** [reply]
Yeah! It works now! Thanks so much.
